PAO-Project/src/Catalog/Catalog_S.java

```diff
@@ -0,0 +1,74 @@
+package Catalog;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class Catalog_S
```

Comment:
[Fix] daca ai un pachet services, aceasta clasa de ce e aici. Also, in java denumirea clasei ar fi CatalogService, fara _ si fara prescurtari de o singura litera